src/os/bluestore/BitAllocator.cc

```diff
@@ -379,18 +379,18 @@ int64_t BitMapZone::get_used_blocks()
 
 bool BitMapZone::reserve_blocks(int64_t num_blocks)
 {
-  alloc_assert(0);
+  alloc_ceph_abort();
```

Comment:
We shall withdraw these changes and change 
https://github.com/ceph/ceph/blob/master/src/os/bluestore/BitAllocator.h#L21 instead.